### PR TITLE
Add ostree-compose and rsync container() calls

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

**Comment by scoheb** [comment]
[test]

**Comment by scoheb** [comment]
[test[

**Comment by scoheb** [comment]
[test]

**Comment by scoheb** [comment]
[test]

**Comment by scoheb** [comment]
[test]

File: Jenkinsfile [hunk 1]
```diff
@@ -38,6 +41,8 @@ properties(
                                 string(defaultValue: '', description: '', name: 'ghprbPullId'),
                                 string(defaultValue: 'stable', description: 'Tag for slave image', name: 'SLAVE_TAG'),
                                 string(defaultValue: 'stable', description: 'Tag for rpmbuild image', name: 'RPMBUILD_TAG'),
+                                string(defaultValue: 'stable', description: 'Tag for rpmbuild image', name: 'RSYNC_TAG'),
```
Comment (johnbieren) [inline Jenkinsfile]: nit pick - description is wrong

File: Jenkinsfile [hunk 2]
```diff
@@ -168,25 +190,31 @@ podTemplate(name: 'fedora-atomic-' + env.ghprbActualCommit,
...
+                        def ostree_props_groovy = "${env.WORKSPACE}/ostree.props.groovy"
                         pipelineUtils.convertProps(ostree_props, ostree_props_groovy)
                         load(ostree_props_groovy)
 
-                        // Teardown resource
-                        pipelineUtils.teardownResources(currentStage)
-
                         // Set our message topic, properties, and content
-                        messageFields = pipelineUtils.setMessageFields("package.complete")
+                        messageFields = pipelineUtils.setMessageFields("compose.complete")
```
Comment (johnbieren) [inline Jenkinsfile]: Good catch on this, we should have a compose.queued message in the previous step too. Nit pick on the comment in the line below this as well.

**Comment by scoheb** [comment]
[merge]

**Pull request merged**